playbooks/roles/ocp-fio/README.md

```diff
@@ -0,0 +1,71 @@
+ocp-fio
+=========
+
+This role is used to deploy File Integrity Operator and run e2e test cases from openshift-test-private repository.
+
+Requirements
+------------
+
+- OCP 4.x healthy cluster
+- OCP secret with name ***podman-secret***  in the default namespace which is used for global secret update and has following keys:
+   ***username***, ***password***  and ***registry***
+
+
+Role Variables
+--------------
+
+| Variable                 | Required | Default          | Comments                                              |
+|--------------------------|----------|------------------|-------------------------------------------------------|
+| fio_enabled              | yes      | false            | Set it to true to run this playbook                   |
```

Comment:
I do not see default value for this variable in defaults file.

Reply:
If Default is false then this variable is not Required.

Reply:
I will add this in default file and make required no.